blueprints/ember-redux/index.js

```diff
@@ -0,0 +1,18 @@
+/*jshint node:true*/
+module.exports = {
+  description: 'Installation blueprint for ember-redux',
+  normalizeEntityName: function() {},
+
+  afterInstall: function() {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-browserify', target: '~1.1.11'},
+      ]
+    }).then(function() {
+      return this.addPackagesToProject([
+        {name: 'redux', target: '~3.5.2'},
```

Comment:
Oh i just realized why we had that show up now (likely) - your first
reducer wasn't correct (remember we had to swap user.js and index.js in
that file because they were reversed?) That would be enough to confused
redux/ likely having that warning show up (if not an error like I'd expect)

Toran

On Thu, Aug 4, 2016 at 8:28 PM, Dustin Farris <notifications@github.com>
wrote:

> In blueprints/ember-redux/index.js
> <https://github.com/toranb/ember-redux/pull/15#discussion_r73630243>:
>
> > @@ -0,0 +1,18 @@
> > +/*jshint node:true*/
> > +module.exports = {
> > +  description: 'Installation blueprint for ember-redux',
> > +  normalizeEntityName: function() {},
> > +
> > +  afterInstall: function() {
> > +    return this.addAddonsToProject({
> > +      packages: [
> > +        {name: 'ember-browserify', target: '~1.1.11'},
> > +      ]
> > +    }).then(function() {
> > +      return this.addPackagesToProject([
> > +        {name: 'redux', target: '~3.5.2'},
>
> No. Please see my response and confirm if you want this change or not.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/toranb/ember-redux/pull/15/files/c2d0055a59eb35c78fdec6ac66a0c321ad2fd03f#r73630243>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAI_0yOlKpQ6NC8S_XoY1FsHc9MDemcnks5qcpGxgaJpZM4JdKi_>
> .
>